static/docs/get-started/add-files.md

```diff
@@ -34,7 +34,7 @@ Git to track versions of your file:
 
 ```dvc
 $ git add data/.gitignore data/data.xml.dvc
-$ git commit -m "add source data to DVC"
+$ git commit -m "add initial data to DVC"
```

Comment:
you can use term `raw` instead

Reply:
Yes! I also thought about "raw" but wasn't sure )  Updated in c632b30